ViteMaDose/Views/CountySelection/Header/CountySelectionHeaderView.swift

```diff
@@ -11,8 +11,8 @@ class CountySelectionHeaderView: UIView {
     @IBOutlet var titleLabel: UILabel!
 
     private enum Constant {
-        static let titleText = "select_your_area".localized()
-        static let highlightedText = "area".localized()
+        static let titleText = LocalizedString.select_your_area
+        static let highlightedText = LocalizedString.area
```

Comment:
Useless constant, you can use `LocalizedString.select_your_area` and `LocalizedString.area` directly instead of `titleText` and `highlightedText`.